src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -427,17 +441,18 @@ export class HuiAreaCard
               if (!(domain in entitiesByDomain)) {
                 return "";
               }
-              return DEVICE_CLASSES[domain].map((deviceClass) =>
-                this._isOn(domain, deviceClass)
-                  ? html`
-                      ${DOMAIN_ICONS[domain][deviceClass]
-                        ? html`<ha-svg-icon
-                            .path=${DOMAIN_ICONS[domain][deviceClass]}
-                          ></ha-svg-icon>`
-                        : ""}
-                    `
-                  : ""
-              );
+              return this._deviceClasses[domain].map((deviceClass) => {
+                const entity = this._isOn(domain, deviceClass);
+                return entity
+                  ? html`<ha-svg-icon
+                      class="alert"
+                      id=${entity.entity_id}
+                      .path=${DOMAIN_ICONS[domain][deviceClass] ||
+                      binarySensorIcon(entity.state, entity)}
+                      @click=${this._moreInfo}
```

Comment:
As a user I would want to know when there are multiple entities that are alerting, and not only the first I think?